x-pack/plugins/infra/public/pages/metrics/layouts/host.ts

```diff
@@ -16,37 +17,54 @@ import {
 export const hostLayoutCreator: InfraMetricLayoutCreator = theme => [
   {
     id: 'hostOverview',
-    label: 'Host',
+    label: i18n.translate('xpack.infra.hostMetrics.hostLabel', {
+      defaultMessage: 'Host',
+    }),
     sections: [
       {
         id: InfraMetric.hostSystemOverview,
         linkToId: 'hostOverview',
-        label: 'Overview',
+        label: i18n.translate('xpack.infra.hostMetrics.overviewSectionLabel', {
```

Comment:
`xpack.infra.metricDetailPage.hostMetricsLayout.overviewSection.sectionLabel` and so on below